dvc/command/update.py

```diff
@@ -35,4 +35,7 @@ def add_parser(subparsers, parent_parser):
     update_parser.add_argument(
         "targets", nargs="+", help="DVC-files to update."
     )
+    update_parser.add_argument(
+        "--rev", nargs="?", help="DVC repository git revision."
```

Comment:
Maybe
```suggestion
        "--rev", nargs="?", help="Git revision in repository to update from."
```
If you agree to this change, please also update it similarly for `dvc import` (ending with `to import from`).